packages/@tinacms/toolkit/src/packages/react-sidebar/components/Nav.tsx

```diff
@@ -19,29 +19,38 @@ limitations under the License.
 import * as React from 'react'
 import { BiExit } from 'react-icons/bi'
 import { FiMoreVertical } from 'react-icons/fi'
-import { ImFilesEmpty } from 'react-icons/im'
 import { VscNewFile } from 'react-icons/vsc'
 import { Menu, Transition } from '@headlessui/react'
-import { SidebarContext, updateBodyDisplacement } from './Sidebar'
+import { updateBodyDisplacement } from './Sidebar'
 import { FormModal } from '../../react-forms'
 import { useEditState } from '@tinacms/sharedctx'
+import type { ScreenPlugin } from '../../react-screens'
 
 interface NavProps {
   children?: any
   className?: string
   userName?: string
+  collections?: { label: string; name: string }[]
   contentCreators?: any
-  screens?: any
+  screens?: ScreenPlugin[]
+  sidebarWidth?: number
+  RenderNavSite: React.ComponentType<{ view: ScreenPlugin }>
+  RenderNavCollection: React.ComponentType<{
+    collection: { label: string; name: string }
+  }>
```

Comment:
The majority of the changes to `Nav` are based around these new props.  I effectively decoupled `Nav` from `SidebarContext`, freeing me from having to maintain `SidebarContext` in the other place I wanted to use `Nav`.

Instead, I pass in the behaviors that I want `Nav` to use.